Sources/Extensions.swift

```diff
@@ -302,3 +303,13 @@ extension UIImage {
         return outPutImageData
     }
 }
+
+extension UIPasteboard {
```

Comment:
I prefer to add a normal property instead of *static* property.

```swift
extension UIPasteboard {
     /// Fetch old text on pasteboard
     var oldText: String? {
        let utf8PlainText = kUTTypeUTF8PlainText as String
        guard let firstItem = items.first else { return nil }
        guard let plainText = firstItem[utf8PlainText] as? String else { return nil }
        return plainText
     }
 }
```